akka-docs/src/main/paradox/stream/operators/Source/completionStage.md

```diff
@@ -4,7 +4,7 @@ Send the single value of the `CompletionStage` when it completes and there is de
 
 @ref[Source operators](../index.md#source-operators)
 
-@@@div { .group-scala }
+@@@div { .group-java }
```

Comment:
In https://github.com/akka/akka/blob/v2.6.3/akka-docs/src/main/paradox/stream/operators/Source/from.md we merged the `apply.md` and `from.md` pages completely, we could consider doing the same for `completionStage`/`future`. Up to you.

Reply:
Will do on a separate PR.

Reply:
raised #28641